test/libethereum/blockchainHelper.cpp

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
why do you call this file blockChainHelper? If it is about testing block chain function you could call it Blockchain.cpp